manager-dashboard/app/views/NewProject/index.tsx

```diff
@@ -326,6 +330,30 @@ function NewProject(props: Props) {
                 }
 
                 const database = getDatabase();
+                const projectTopicKey = projectTopic?.toLowerCase() as string;
+
+                const projectRef = databaseRef(database, 'v2/projects/');
+                const prevProjectNameQuery = query(
+                    projectRef,
+                    orderByChild('projectTopicKey'),
+                    equalTo(projectTopicKey),
+                );
```

Comment:
Let' also query the draft projects. We might need to also update the firebase rules.

Reply:
Let's add a NOTE that we should also check draft projects.